contrib/test/crio-integration-playbook.yaml

```diff
@@ -57,6 +57,7 @@
       name: "{{ item }}"
       state: latest
     with_items:
+      - iptables
```

Comment:
I am curious. Did we change the image so we now have to install iptables?

Reply:
It was part of that "hidden" playbook, which is a terrible place to put dependencies.  They all need to be spelled out here, otherwise the tests are beholden to some external entity "getting it right".  In the interest of time, I can copy paste all of them here, then y'all can weed out the ones you don't need, later.

Reply:
okay